### DISCUSSION TITLE
How to use the PNAconv?

### DISCUSSION THREAD

**Discussion by StefanIsSmart** [pyg-team/pytorch_geometric#8775] [General]
When I want to use the PNAConv, I found that I need to put in the deg of the graph for init the Conv. Layer.
But how to know the deg of different graphs in the different batches? 
I don't know how to make sure the value of deg.
Maybe put in it in the forward process is better?

**Comment by rusty1s** [comment]
`deg` refers to a global statistics across all training graphs. You can look at the `PNAConv.get_degree_histogram` for an example on how to compute it.

**Comment by chemousesi** [comment]
Check this example on how to use PNA
https://github.com/pyg-team/pytorch_geometric/blob/master/examples/pna.py
